### PR TITLE
Steven Pizarro Portfolio Project 

### PR DESCRIPTION
**Pull request by stevenp19-zz**

### CODE REVIEW

File: html5up-story/index.html [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+						<div class="image">
+							<img src="images/phoeniximage.jpg" alt="phoenix skyline" />
+						</div>
+					</section>
+
+				<!-- Two -->
+					<section class="spotlight style1 orient-right content-align-left image-position-center onscroll-image-fade-in" id="first">
+						<div class="content">
+							<h2>Sweet Eats Bakery</h2>
+							<p>This project was a study on html and css.</p>
```
Comment (ckopecky) [inline html5up-story/index.html]: You might say 

```Technology Used: HTML, CSS```

File: html5up-story/index.html [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+						<div class="image">
+							<img src="images/cupcake.jpg" alt="cupcake" />
+						</div>
+					</section>
+
+				<!-- Three -->
+					<section class="spotlight style1 orient-left content-align-left image-position-center onscroll-image-fade-in">
+						<div class="content">
+							<h2>Great Idea</h2>
+							<p>Here is a project where we were working on the box model in CSS.</p>
```
Comment (ckopecky) [inline html5up-story/index.html]: ```Technology Used: HTML, CSS/Box Model```

File: html5up-story/index.html [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+								<li><a href="#" class="button">View Code</a></li>
+							</ul>
+						</div>
+						<div class="image">
+							<img src="images/greatideasplash.jpg" alt="" />
+						</div>
+					</section>
+
+				<!-- Four -->
+					<section class="spotlight style1 orient-right content-align-left image-position-center onscroll-image-fade-in">
```
Comment (ckopecky) [inline html5up-story/index.html]: Let's go ahead and get rid of the Latin that doesn't pertain to you.

**Review by ckopecky — CHANGES_REQUESTED** [review]

File: html5up-story/index.html (hunk 1 above)
Comment (stevenp19-zz) [reply html5up-story/index.html]: ok, that sounds better.

**Comment by stevenp19-zz** [comment]
ok ill go in vscode now to fix it!

**Comment by stevenp19-zz** [comment]
hopefully, I did this right.

**Pull request merged**